WordPress/WordPressNotificationContentExtension/Sources/NotificationContentView.swift

```diff
@@ -0,0 +1,171 @@
+import UIKit
+
+import WordPressShared
+import WordPressUI
+
+// MARK: - NotificationContentView
+
+/// This class is responsible for presenting the content of a rich notification within the "Long Look".
+/// It consists of an avatar view, "noticon", subject, & body.
+///
+class NotificationContentView: UIView {
+
+    // MARK: Properties
+
+    private struct Metrics {
+        static let avatarDimension  = CGFloat(46)
+        static let noticonOffset    = CGFloat(19)
+        static let noticonDimension = CGFloat(24)
+        static let noticonFontSize  = CGFloat(17)
+        static let subviewSpacing   = CGFloat(12)
+    }
+
+    private struct Styles {
+        /// Extracted from https://bettertogethermobile.wordpress.com/2018/06/25/ios12-rich-er-notification-design-v1-5/ via Digital Color Meter
+        static let noticonBackgroundColor = UIColor(red: (82/255.0), green: (180/255.0), blue: (231/255.0), alpha: 1.0)
```

Comment:
We should match this to a colour from `WPStyleGuide` – I think it actually looks like `lightBlue`: `(120, 220, 250)`, but perhaps @nozomimimi could confirm.

Reply:
After speaking to @nozomimimi, this should match whatever we use in the Notifications list in WPiOS.